cmds/admin_server/storage/storage.go

```diff
@@ -17,10 +19,10 @@ var (
 )
 
 type Storage interface {
-	StoreLog(Log) error
-	GetLogs(Query) (*Result, error)
+	StoreLog(xcontext.Context, Log) error
```

Comment:
since this interface is supposed to be self-documenting (as it's missing comments), id add names for params
```suggestion
	StoreLog(ctx xcontext.Context, entry Log) error
```